trello_items.py

```diff
@@ -0,0 +1,89 @@
+import requests
+
+from flask_config import Config
+from item import Item
+
+
+def get_items():
+    """
+    Fetches all saved items from the session.
+
+    Returns:
+        list: The list of saved items.
+    """
+    r = requests.get(f'https://api.trello.com/1/lists/{Config.LIST_ID}/cards?key={Config.KEY}&token={Config.TOKEN}')
+    done_request = requests.get(
+        f'https://api.trello.com/1/lists/{Config.DONE_LIST_ID}/cards?key={Config.KEY}&token={Config.TOKEN}')
+    response = r.json()
```

Comment:
Variable names are a bit off:

`r` is a response and `r.json()` is the response body.
`done_request` is actually a Response object.

More importantly, it's a bit confusing how you are treating the two requests slightly differently when they are doing the same thing for different lists. Would be neater to have something like:
```
not_started_response = requests.get(...)
done_response = requests.get(...)
```